module/platform/Webots/src/Webots.cpp

```diff
@@ -391,6 +391,15 @@ namespace module::platform {
                 servo.present_position = 0.0;
                 servo.present_speed    = 0.0;
             }
+
+            auto targets = std::make_unique<ServoTargets>();
+
+            for (int i = 0; i < 20; i++) {
```

Comment:
we probably shouldn't hard code 20

Reply:
Can you do this, not sure how to get size :')

Reply:
```suggestion
            for (int i = 0; i < ServoID::NUMBER_OF_SERVOS; i++) {
```
I think this is what we want.
Might need to include ServoID in the file as well, but `Webots.cpp` might already have it.